.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,24 +1,31 @@
-<!-- Describe your pull request here -->
+**Describe your pull request here**
 
 ----
 
 Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).
 
-* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title
-* New recipes not directly relevant to the biological sciences need to be submitted to [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda
-* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please add the `please review & merge` label.
+* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
+* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
+* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
```

Comment:
I think adding a label is only possible for members, right (didn't check)?
`@BiocondaBot please add label` can be used by anyone :).